frontend/src/features/discussion/ui/DiscussionRank/RankContents.tsx

```
@@ -0,0 +1,35 @@
+    >
+      <Flex
+        className={rankContainerStyle}
+        gap={200}
+        width='100%'
+      >
+        {data.slice(0, TOP_CARD_NUM).map((discussion, idx) => 
+          <DiscussionCard
+            discussion={discussion}
+            rank={idx + 1}
+            size='lg'
+          />)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)